proto/opamp.proto

```diff
@@ -394,6 +419,31 @@ message OtherConnectionSettings {
     // Other connection settings. These are Agent-specific and are up to the Agent
     // interpret.
     map<string, string> other_settings = 4;
+
+    // Optional connection specific TLS settings.
+    // Status: [Development]
+    TLSConnectionSettings tls = 5;
+}
+
+// Status: [Development]
+message TLSConnectionSettings {
+  // Provides CA cert contents as a string.
+  string ca_pem_contents = 1;
+
+  // Load system CA pool alongside any passed CAs.
+  bool include_system_ca_certs_pool = 2;
```

Comment:
This doesn't seem to be needed for the new flow.

Reply:
For the flow, no.
Do we want the settings to only support the flow, or to correspond more with general TLS settings?
I think having more generic settings present will help in the future

Reply:
We need to have specific use cases in mind that require these settings. My preference is to avoid adding settings that don't have clear uses and important cases.